spyderlib/plugins/inspector.py

```diff
@@ -199,6 +199,7 @@ def setup_page(self):
         self.setLayout(vlayout)
 
 
+

```

Comment:
Please remove this empty blank line. It's not needed because there are already two blanks before it :-)